Playbooks/playbook-Convert_file_hash_to_corresponding_hashes_.yml

```
@@ -0,0 +1,814 @@
+id: Convert file hash to corresponding hashes
+version: -1
+name: 'Convert file hash to corresponding hashes '
+fromversion: 4.0.0
```

shouldn't this be 4.5.0?
```suggestion
fromversion: 4.5.0
```